CHANGES.rst

```diff
@@ -1,3 +1,12 @@
+2.0.3 (2023-07-10)
```

Comment:
2.0.4. Keep release date the same, we will pick release date once release manager approves.